src/components/location-search/location-search.tsx

```diff
@@ -134,7 +141,7 @@ const LocationSearch: React.FC<LocationSearchProps> = ({
 					</div>
 				)}
 			</div>
-			<div className="w-[15%] max-w-[15%] flex flex-col justify-start items-center mt-1 mr-1 lg:mr-0">
+			<div className="min-w-[10%] h-full flex flex-col justify-start items-center mt-1 mr-1 lg:mr-0">
```

Comment:
I would make the button the same height as the searchbar: 
<div className="min-w-[10%] flex flex-col mr-1 lg:mr-0">